src/main.cc

```diff
@@ -1830,6 +1916,12 @@ watch_child(char *argv[])
 
     int nullfd;
 
+    // XXX: zero values are not supported because they result in
```

Comment:
I suggest replacing this `XXX` label with `TODO`. A slight restriction on configuration values does not represent a bug IMO. Eventually, somebody may find a way to support them and address this TODO without breaking any existing configurations (because we prohibit zero values).